common/scylla_create_devices

```diff
@@ -104,6 +107,9 @@ if __name__ == "__main__":
     parser.add_argument('--data-device', dest='data_device', action='store',
                         choices=["auto", "attached", "instance_store"],
                         help='Define type of device to use for scylla data: attached|instance_store')
+    parser.add_argument('--raid-level', dest='raid_level', action='store',
+                        choices=["0", "5"],
```

Comment:
type should be int, you check in line 44 integers

Reply:
anyway, the code in lines 44-46 is redundant, so for consistency matters use int or str

Reply:
fixed